.gitignore

```diff
@@ -8,3 +8,4 @@ inst/doc
 /doc/
 /Meta/
 scratch.R
+pkgdown/
```

Comment:
This folder can contain important files we want to commit. Do you have any issues that lead you to add it here?

Reply:
Currently it only contains the directoy `favicon/` with icons of different sizes - should these be committed? I can allow tracking if so.

Reply:
Converting this to an issue for now - can be added to tracking if it becomes necessary, but I don't see a `pkgdown` directory in our other repos so keeping it out for now.